backend/src/slide/slide.controller.ts

```diff
@@ -1,15 +1,27 @@
-import { Controller, Get, Post, Body, Patch, Param, Delete } from '@nestjs/common';
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+} from '@nestjs/common';
 import { SlideService } from './slide.service';
-import { CreateSlideDto } from './dto/create-slide.dto';
-import { UpdateSlideDto } from './dto/update-slide.dto';
+import { CreateSlideDTO } from '@meethead/types';
+import { UpdateSlideDTO } from './dto/update-slide.dto';
```

Comment:
Why is `UpdateSlideDTO` imported from `dto` folder? I guess it should be imported from `@meethead/types`

Reply:
We need this to make attributes optional using a helper from Nestjs. This is why UpdateSlideDto is defined here